backend/tests/common.py

```diff
@@ -735,11 +758,29 @@ def fetch_alert_rule(
 
         return self._assert_alert_rule(response.json())
 
+    def fetch_model_alert_rule(
```

Comment:
similarly, maybe rename it to `fetch_data_ingestion_alert_rule`?

usual alert rules also exist within the context of a model, therefore this name is a little bit confusing and inconsistent with other parts of the code base